src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package tasklist;
+
+import java.util.ArrayList;
+
+import task.Task;
+import ui.Ui;
+
+/**
+ * Class to encapsulate the list of tasks and its corresponding methods.
+ */
+public class TaskList {
+    private static ArrayList<Task> taskList;
+
+    /**
+     * Constructor that uses given ArrayList to create a TaskList object.
+     *
+     * @param list ArrayList to be modelled after.
+     */
+    public TaskList(ArrayList<Task> list) {
+        this.taskList = list;
+    }
+
+    /**
+     * Constructor for a new TaskList object.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
```

Comment:
I like the use of method overloading to make the construction of TaskLists easier.